samples/drivers/led_strip/src/main.c

```diff
@@ -19,17 +19,20 @@ LOG_MODULE_REGISTER(main);
 #include <zephyr/drivers/spi.h>
 #include <zephyr/sys/util.h>
 
-#define STRIP_NODE		DT_ALIAS(led_strip)
+#define STRIP_NODE DT_ALIAS(led_strip)
 
 #if DT_NODE_HAS_PROP(DT_ALIAS(led_strip), chain_length)
-#define STRIP_NUM_PIXELS	DT_PROP(DT_ALIAS(led_strip), chain_length)
+#define STRIP_NUM_PIXELS DT_PROP(DT_ALIAS(led_strip), chain_length)
 #else
 #error Unable to determine length of LED strip
 #endif
 
 #define DELAY_TIME K_MSEC(CONFIG_SAMPLE_LED_UPDATE_DELAY)
 
-#define RGB(_r, _g, _b) { .r = (_r), .g = (_g), .b = (_b) }
+#define RGB(_r, _g, _b)                                                                            \
+	{                                                                                          \
+		.r = (_r), .g = (_g), .b = (_b)                                                    \
+	}
```

Comment:
Please remove all the indentation changes on this file. They are irrelevant for the ws2812-uart driver. If you want to fix the `led_strip` sample then do it with a separate commit and eventually another PR. As a rule of thumb, don't mix several changes in a single commit.

Following this rule, you should consider splitting your commit into three:

- addition of the `ws2812-uart` DT binding
- addition of the `ws2812-uart` driver
- update of the `led_strip` sample